src/Runner.Worker/Handlers/CompositeActionHandler.cs

```diff
@@ -273,30 +279,7 @@ private async Task RunStepAsync(IStep step)
                 step.ExecutionContext.Result = Common.Util.TaskResultUtil.MergeTaskResults(step.ExecutionContext.Result, step.ExecutionContext.CommandResult.Value);
             }
 
-            // Fixup the step result if ContinueOnError.
-            if (step.ExecutionContext.Result == TaskResult.Failed)
```

Comment:
We don't need this since we'll explicitly disallow Continue on error for each composite step for now.